02-peer-protocol.md

```diff
@@ -2095,6 +2238,49 @@ fall-behind detection.  An implementation can offer both, however, and
 fall back to the `option_data_loss_protect` behavior if
 `option_static_remotekey` is not negotiated.
 
+## Appendix A: The lease fee (`option_will_fund`)
+
+The lease fee is added to the accepter's balance in a channel, in addition
+to the `funding_satoshi` that they are contributing. The channel initiator
+must contribute enough funds to cover `open_channel2`.`funding_satoshis`,
+the lease fee, and their tx weight * `funding_feerate_perkw` / 1000.
+
+The lease fee is calculated as:
+
+	`funding_fee_base_sat` +
+	`accept_channel2`.`funding_satoshis` * `funding_fee_proportional_basis` / 1000 +
+	`funding_weight` * `funding_feerate_perkw` / 1000
+
+E.g.
+An node requests 100.0000sats at a feerate of 2500perkw. They
+are contributing 50.0000sats. Their weight contribution to the
```

Comment:
4 decimal point is particularly weird? but the decimals here are really confusing if you're from Europe and read this ias 100,000 sats!

Reply:
As a European, I got used to interpreting `.` as a decimal point in programming-related texts. Can't say about others but I'd be surprised if there are many people who didn't do similarly. Personally, I get sometimes confused in Europe actually. :D